app/src/modules/settings/routes/flows/triggers.ts

```diff
@@ -227,6 +227,28 @@ export function getTriggers() {
 						default_value: 'GET',
 					},
 				},
+				{
```

Comment:
We should move this after the `async` option, currently to much content shift when async is selected

Reply:
Done via d91c536a86032f9f9008791670691a71a8d86acc